depmapomics/copynumbers.py

```diff
@@ -303,6 +362,19 @@ def postProcess(
         i["hgnc_symbol"] + " (" + str(i["entrezgene_id"]).split(".")[0] + ")"
         for _, i in mybiomart.iterrows()
     ]
+    mybiomart = mybiomart[
+        ~mybiomart.entrezgene_id.isna()
+    ]  # dropping all nan entrez id cols
+    # drop Ychrom if > maxYchrom
+    ychrom = segments[segments.Chromosome.str.contains("Y")]
+    countYdrop = [
+        i
+        for i in set(ychrom[SAMPLEID])
+        if len(ychrom[ychrom[SAMPLEID] == i]) > maxYchrom
+    ]
+    segments = segments[
+        ~((segments[SAMPLEID].isin(countYdrop)) & (segments.Chromosome == "Y"))
+    ]
     genecn = mut.toGeneMatrix(mut.manageGapsInSegments(segments), mybiomart)
     # validation step
     print("summary of the gene cn data:")
```

Comment:
@jkobject currently `mut.checkGeneChangeAccrossAll(genecn, thresh=genechangethresh)` (two lines below) isn't doing or returning anything here, and I wonder if it had been run in any previous quarter. Do we want to get the genes that have low variance across the lines and remove them from the matrices? Should we add it here?